.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test
+  - dartanalyzer
```

Comment:
Only run against the dev analyzer.

Why aren't you using [the `.travis.yml` file from `oss_project_templates`](https://github.com/dart-lang/oss_project_templates/blob/master/dart-github-dart-lang/.travis.yml)?